### PR TITLE
Mail2kvanitha/c01-git01

### PR DESCRIPTION
**Pull request by mail2kvanitha**
# Description

Exercise (e.g. C01-E01): c01-git01

Student Name: Vanitha Kaliyaperumal

> A brief explanation of what has been done.
Understanding of git commands how to merge, rebase, revert and reset.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# c01-git01
+
+## Commands executed
+
+1. Commands typed to solve the "Merging" exercise
+git checkout master
+git merge <branch-name>
+git checkout <branch-name>
+git merge master
```
Comment (kikobr82) [inline classes/01class/exercises/c01-git01/mail2kvanitha/README.md:9]: Adjust formatting..

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```
+
+2. Commands typed to solve the "Rebasing" exercise
+git checkout -b branch-name
+git commit
+git checkout master
+git commit
+git checkout <branch_name>
+git rebase master
```
Comment (kikobr82) [inline classes/01class/exercises/c01-git01/mail2kvanitha/README.md:19]: Adjust formatting

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+git commit
+git checkout <branch_name>
+git rebase master 
+
+```
+
+3. Commands typed to solve the "Revert and reset" exercise
+git reset master
+git checkout pushed
+git revert pushed
```
Comment (kikobr82) [inline classes/01class/exercises/c01-git01/mail2kvanitha/README.md:26]: adjust formatting

File: classes/01class/exercises/c01-git02/mail2kvanitha/c01-git02.txt [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# c01-git02
```
Comment (kikobr82) [inline classes/01class/exercises/c01-git02/mail2kvanitha/c01-git02.txt:1]: This file should not be part of the c01-git01 exercise delivery

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md (hunk 3 above)
Comment (mail2kvanitha) [reply classes/01class/exercises/c01-git01/mail2kvanitha/README.md:26]: Adjusted formatting, Please check and confirm if anything pending.

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md (hunk 1 above)
Comment (mail2kvanitha) [reply classes/01class/exercises/c01-git01/mail2kvanitha/README.md]: projects/project02/application/web/public/favicon.ico file removed from projects

File: classes/01class/exercises/c01-git01/mail2kvanitha/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+Commands typed to solve the "Merging" exercise
+git checkout master
+git merge branch-name
+git checkout branch-name
+git merge master
+
+```
+
+Commands typed to solve the "Rebasing" exercise
+git checkout -b branch-name
```
Comment (denstorti) [inline classes/01class/exercises/c01-git01/mail2kvanitha/README.md:14]: that's correct, the exercise uses `bugFix` for the `branch-name` but the result is the same 👍

**Pull request merged**